### DISCUSSION TITLE
Java - error: package nitf does not exit

### DISCUSSION THREAD

**Discussion by s2000coder** [mdaus/nitro#339] [Q&A] [answered]
```
javac ReadImageSample.java 
ReadImageSample.java:6: error: package nitf does not exist
import nitf.IOHandle;
           ^
ReadImageSample.java:7: error: package nitf does not exist
import nitf.ImageReader;
           ^
ReadImageSample.java:8: error: package nitf does not exist
import nitf.ImageSegment;
           ^
ReadImageSample.java:9: error: package nitf does not exist
import nitf.ImageSubheader;
```
This is on CentOS with OpenJDK 1.8.0.

The `javac` compiler can't find the nitf library. Nitro is successfully built on the OS.

Setting the path to where the `libnitf.jni-c.so` is located doesn't do anything.
`export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/path/to/nitro/lib`

Any idea on how to run the java file?

**Comment by JDanielSmith** [comment, marked as answer]
From [README.md](https://github.com/mdaus/nitro/blob/master/README.md) "The MATLAB and JAVA layers have not been touched in years; they are no longer built, code remains in the **archive** directory."
